README.md

```diff
@@ -117,8 +117,8 @@ submodule.
 Here's a look at all the git submodules that are currently tracked in
 the rocket-chip repository:
 
-* **chisel2**
-([https://github.com/ucb-bar/chisel](https://github.com/ucb-bar/chisel)):
+* **chisel3**
+([https://github.com/ucb-bar/chisel3](https://github.com/ucb-bar/chisel3)):
 At Berkeley, we write RTL in Chisel. For those who are not familiar
 with Chisel, please go take a look at
 [http://chisel.eecs.berkeley.edu](http://chisel.eecs.berkeley.edu). We
```

Comment:
Does this link apply to chisel3? One of the main things you find at that link is the chisel-tutorial, is the chisel-tutorial now in chisel3?